HarpGuru/HarpFace/HarpCell/components/index.tsx

```diff
@@ -20,14 +20,15 @@ import { ExperienceModes } from '../../../helpers/setGlobalReactNState'
 export const HarpCell = (props: HarpCellProps): React.ReactElement => {
   const [activeHarpStrata, setActiveHarpStrata] = useGlobal('activeHarpStrata')
   const [activeExperienceMode] = useGlobal('activeExperienceMode')
+  const [activeDisplayMode] = useGlobal('activeDisplayMode')
   const [quizQuestion] = useGlobal('quizQuestion')
   const harpCellProps = {
```

Comment:
The name of this variable is clearly wrong now since it contains a HarpCellProps spread in to it. It's been wrong for a while, but I'm acknowledging it again to make sure it is covered off in an upcoming change.

Reply:
Referenced in open issue: https://github.com/js-jslog/harpnative/issues/45#issuecomment-663618566